.github/CODEOWNERS

```diff
@@ -8,11 +8,11 @@
 *                       @googleapis/yoshi-java @googleapis/cloud-native-db-dpes @googleapis/api-datastore-sdk @googleapis/api-firestore-partners
 
 # for handwritten libraries, keep codeowner_team in .repo-metadata.json as owner
-**/*.java               @googleapis/cloud-native-db-dpes @googleapis/api-datastore-sdk @googleapis/api-firestore-partners
+**/*.java               @googleapis/yoshi-java @googleapis/cloud-native-db-dpes @googleapis/api-datastore-sdk @googleapis/api-firestore-partners
```

Comment:
Can we use `cloud-java-team-teamsync` instead? We should start deprecating `yoshi-java`.

Reply:
Done